src/morphoclass/data/morphology_dataset.py

```diff
@@ -493,72 +491,3 @@ def guess_layer(self):
                 return None
         else:
             return None
-
-    def to_lowerdim_dataset(self, embedding_type, **feat_extr_kwargs):
```

Comment:
The only place where this function was used was in the submodules
- `morphoclass.feature_extractors.pyramidal_cells`
- `morphoclass.feature_extractors.interneurons`

Since these two submodules are removed in this PR, we can safely remove this function.